packages/fxa-settings/src/pages/ResetPassword/ConfirmTotpResetPassword/index.tsx

```diff
@@ -130,18 +134,35 @@ const ConfirmTotpResetPassword = ({
               codeErrorMessage,
               setCodeErrorMessage,
               viewName: 'confirm-totp-reset-password',
+              gleanDataAttrs: {
+                id: 'reset_password_confirm_totp_code_submit_button',
+              },
             }}
           />
-          <div className="mt-5 link-blue text-sm flex justify-end">
-            <FtlMsg id="confirm-totp-reset-password-trouble-code">
-              <button
-                onClick={() => {
-                  setShowRecoveryCode(true);
-                }}
-              >
+          <div className="mt-5 flex justify-between items-center">
+            <button
+              className="link-blue text-sm"
+              data-glean-id="reset_password_confirm_totp_use_different_account_button"
+              onClick={() => {
+                // Navigate to email first page and keep search params
+                hardNavigate('/', {}, true);
+              }}
```

Comment:
hmm I actually don't know how the Glean automatic event behaves here.  Hopefully Glean manages to send the event before the navigation?